### ISSUE TITLE
The library does not transform the keywords of the programming language in any way

### ISSUE THREAD

**Issue by IgnatLomkov** [sequelize/sequelize-auto#570]
Suppose I have a **"const"** table in SQL, and if I write it down in the model, then the word **"const"** will remain unchanged, and as a result, the program will give the error **"Unexpected token 'const'"** when it starts, which is logical in principle. I already solved this problem by naming the table **"constTable"** and now everything works well, but I wanted sequelize-auto to do something similar on its own at the table naming level

**Comment by steveschmitt** [comment]
Yes, we should escape/transform names that are JavaScript keywords.  What should be the standard way of escaping?

**Comment by IgnatLomkov** [comment]
I think standart escape should be something like original name + original object type. For example, tables should be something like this:
```js
const originalObjectName = 'const', objectType = 'Table'
const totalName = `${originalObjectName}${objectType}`
console.log(totalName)
```

**Comment by steveschmitt** [comment]
Fixed in sequelize-auto@0.8.7
